src/frontend/test_runner/tests/testdata/agg.yaml

```diff
@@ -673,14 +678,17 @@
     select 1 from a having exists(
       select count(b1) filter (where min(a1) < 3) from b
     );
-  binder_error: 'Invalid input syntax: aggregation function in filter clause is not
-    supported'
+  binder_error: |-
+    Feature is not yet implemented: aggregation function in filter clause
+    No tracking issue yet. Feel free to submit a feature request at https://github.com/risingwavelabs/risingwave/issues/new?labels=type%2Ffeature&template=feature_request.yml
 - sql: |
     /* agg filter: table function */
     /* This case is NOT valid in PostgreSQL */
     create table a (a1 int, a2 int);
     select count(a1) filter (where unnest(array[1]) < 1) from a;
-  binder_error: 'Invalid input syntax: table function in filter clause is not supported'
+  binder_error: |-
+    Feature is not yet implemented: table function in filter clause
+    No tracking issue yet. Feel free to submit a feature request at https://github.com/risingwavelabs/risingwave/issues/new?labels=type%2Ffeature&template=feature_request.yml
```

Comment:
Table function is invalid instead of not implemented? 😂

Reply:
Based on my current test cases, yes. But I am not sure whether there are ways to construct a valid query that looks like "table function in filter clause" in the AST. Just like how we thought nested aggs are invalid, except that when the inner agg is correlated and belongs to the outer query.